sabnzbd/deobfuscate_filenames.py

```diff
@@ -131,16 +132,21 @@ def is_probably_obfuscated(myinputfilename):
     return True  # default not obfuscated
 
 
-def deobfuscate_list(filelist, usefulname):
+def deobfuscate_list(filelist, usefulname, extension_too=True):
```

Comment:
You added `extension_too` but except for tests not using it anywhere, so why add it?